src/main/java/io/anserini/search/SearchArgs.java

```diff
@@ -101,6 +101,9 @@
       "the top documents from the initial round ranking.")
   public int rerankcutoff = 50;
 
+  @Option(name = "-rfQrels", metaVar = "[file]", usage = "qrels file used for relevance feedback")
```

Comment:
How about just `-qrels`? Is rf redundant?
Dunno... up for discussion.

Reply:
I add `rf` for two reasons: 

- it is future proof, in case there is another need for qrels
- it is more meaningful and shows what it is for.

But I do not have a strong opinion on this. If you think it is unnecessary, I will change it to `-qrels`

Reply:
re: future proofing, then `-rf.qrels`? Just like how we have `-bm25.b`? I'll leave the choice to you...

Reply:
I feel `-rfQrels` is better since we do not have `-bm25` corresponding to `-bm25.b`?

Reply:
According to the discussion above, `-rf.qrels` would be better here since we will have the relevance grade parameter in the future.